src/vs/server/node/remoteTerminalChannel.ts

```diff
@@ -69,7 +69,7 @@ class CustomVariableResolver extends AbstractVariableResolverService {
 			getLineNumber: (): string | undefined => {
 				return resolvedVariables['lineNumber'];
 			}
-		}, undefined, Promise.resolve(env));
+		}, undefined, undefined, Promise.resolve(env));
```

Comment:
How do we pass the `pathService` to this file too? Do we change this constructor as well?

Reply:
Yes, you'll need to change the constructor.

Reply:
I've added the paths to the contructors, now it seems I need to implement the service in  `instantiationService.ts`.

Also, it seems I need to initiate the service in this line. I don't know what method I would do this:

 https://github.com/microsoft/vscode/blob/eb0935b068f69199647859a3658da4eeef115990/src/vs/server/node/serverServices.ts#L190